packages/node/src/config.js

```diff
@@ -19,6 +19,10 @@ module.exports = {
     ...schema.logger,
     defaultValue: () => getPrefixedConsole()
   },
+  autoBreadcrumbs: {
+    ...schema.autoBreadcrumbs,
+    defaultValue: () => false
```

Comment:
This change is required so that _every_ notify() doesn't also cause the warning log due to this:
https://github.com/bugsnag/bugsnag-js/blob/b089283dcc27ffd18871fa308b465cbe87db06c5/packages/core/client.js#L215-L222

Reply:
Can autoBreadcrumbs be removed as a config option instead?

Reply:
The only drawback of that is it doesn't look like it's possible to remove that property from the TypeScript definition of the options interface